package.json

```diff
@@ -76,6 +80,7 @@
     "graphql-cli": "2.15.8",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "3.0.6",
+    "husky": "^0.14.3",
```

Comment:
why this? :)

Reply:
It's used to set up git hooks from `package.json`. It's the official suggestion from commitlint's documentation. Here's the git hook config that husky is used for https://github.com/kiwicom/smart-faq/blob/bc04ee40c20f35602d1e83b22a471cff04c08e4d/package.json#L33